app/controllers/placements/support/providers_controller.rb

```diff
@@ -3,6 +3,10 @@ def new
     @provider_form = ProviderOnboardingForm.new
   end
 
+  def show
+    @provider = Provider.placements.find(params.require(:id)).decorate
```

Comment:
I think we can also do this. Not sure if we have a preference?

```suggestion
    @provider = Placements::Provider.find(params.require(:id)).decorate
```